cmd/kubeadm/app/componentconfigs/registrations.go

```diff
@@ -96,15 +92,14 @@ var Known Registrations = map[RegistrationKind]Registration{
 		// TODO: When a beta version of the kube-proxy ComponentConfig API is available, start using it
 		MarshalGroupVersion: kubeproxyconfigv1alpha1.SchemeGroupVersion,
 		// AddToSchemeFuncs must use v1alpha1scheme defined in k8s.io/kubernetes, because the schema defined in k8s.io/kube-proxy doesn't have defaulting functions
-		AddToSchemeFuncs: []AddToSchemeFunc{kubeproxyconfig.AddToScheme, kubeproxyconfigv1alpha1scheme.AddToScheme},
+		AddToSchemeFuncs: []AddToSchemeFunc{kubeproxyconfigv1alpha1.AddToScheme},
 		DefaulterFunc:    DefaultKubeProxyConfiguration,
-		ValidateFunc:     ValidateKubeProxyConfiguration,
-		EmptyValue:       &kubeproxyconfig.KubeProxyConfiguration{},
+		EmptyValue:       &kubeproxyconfigv1alpha1.KubeProxyConfiguration{},
 		GetFromInternalConfig: func(cfg *kubeadmapi.ClusterConfiguration) (runtime.Object, bool) {
 			return cfg.ComponentConfigs.KubeProxy, cfg.ComponentConfigs.KubeProxy != nil
 		},
 		SetToInternalConfig: func(obj runtime.Object, cfg *kubeadmapi.ClusterConfiguration) bool {
```

Comment:
do these methods need a rename as they are setting to public type now?

Reply:
hm, actually they are setting the public types in the internal kubeadm config so it's fine.